content/docs/1.4.0/volumes-and-nodes/create-volumes.md

```diff
@@ -48,6 +50,8 @@ When the Pod is deployed, the Kubernetes master will check the PersistentVolumeC
     #  ]'
     ```
 
+    In particular, starting with v1.4.0, the parameter `mkfsParams` can be used to specify filesystem format options for each StorageClass and the options will override the global settings.
```

Comment:
There are no `the global settings` to be overridden, right? I just removed that.

```suggestion
    In particular, starting with v{{< current-version >}}, the parameter `mkfsParams` can be used to specify filesystem format options for each StorageClass.
```

Reply:
@weizhe0422 I didn't see you change this?